extensions/blocks/social-previews/constants.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+export const AVAILABLE_SERVICES = [
+	{
+		title: __( 'Google Search' ),
```

Comment:
You'll need to add the `jetpack` textdomain here and below.

Reply:
Yeah, I've already noticed in my followup PR but I'll backport it here too.